C-Plus-Plus/graphs/DFS.cpp

```diff
@@ -0,0 +1,91 @@
+/*Depth first search (DFS) algorithm starts with the initial node of the graph G, 
+and then goes to deeper and deeper until we find the goal node or the node which has no children. 
+The algorithm, then backtracks from the dead end towards the most recent node that is yet to be completely unexplored. */
+
+#include<bits/stdc++.h>
+using namespace std;
+//nodes 1 to n
+void createAdjList(vector<int> adj[], vector<vector<int>> &edges)
+{
+    int m = edges.size();
+    for(int i=0; i<m; i++)
+    {
+        adj[edges[i][0]].push_back(edges[i][1]);
+        adj[edges[i][1]].push_back(edges[i][0]);
+
+    }
+}
+
```

Comment:
Add a single line comment to explain what a function does, this will help beginners to get a clear idea of your approach.